_posts/2017-07-24-so-you-need-to-hire-good-engineers.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+id: so-you-need-to-hire-good-engineers
+title: "So You Need To Hire Good Engineers"
+date: 2017-07-24 07:46:00
+authors: [rachel-lee]
+categories: [Engineering]
+tags: [Hiring]
+comments: true
+cover_photo: /img/so-you-need-to-hire-good-engineers/cover.jpg
+excerpt: "If you are in a fast growing tech startup, you’re probably actively interviewing and hiring engineers to scale teams. My question to you is, what hiring strategy are you using when interviewing engineering warriors?"
```

Comment:
I would do a search and replace to convert all `’` into `'`,  `“` into `"` and so on.